test/browser_tests/spec_suites/organisms/header.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var breakpointsConfig = require( BASE_JS_PATH + 'config/breakpoints-config' );
+
+describe( 'Header', function() {
+  var BASE_SEL = '.o-header';
+  var LOGO_SEL = BASE_SEL + '_logo-img';
+  var MEGA_MENU_SEL = BASE_SEL + ' .o-mega-menu';
+  var GLOBAL_SEARCH_SEL = BASE_SEL + ' .m-global-search';
+  var GLOBAL_CTA_LG_SEL = BASE_SEL + ' .m-global-header-cta__horizontal';
+  var GLOBAL_CTA_SM_SEL = BASE_SEL + ' .m-global-header-cta__list';
+  var GLOBAL_EYEBROW_LG_SEL = BASE_SEL + ' .m-global-eyebrow__horizontal';
+  var GLOBAL_EYEBROW_SM_SEL = MEGA_MENU_SEL + ' .m-global-eyebrow__list';
+
+  var _dom;
+
+  beforeAll( function() {
+    _dom = {
+      header:            element( by.css( BASE_SEL ) ),
+      logo:              element( by.css( LOGO_SEL ) ),
+      megaMenu:          element( by.css( MEGA_MENU_SEL ) ),
+      globalSearch:      element( by.css( GLOBAL_SEARCH_SEL ) ),
+      globalHeaderCtaLG: element( by.css( GLOBAL_CTA_LG_SEL ) ),
+      globalHeaderCtaSM: element( by.css( GLOBAL_CTA_SM_SEL ) ),
+      globalEyebrowLG:   element( by.css( GLOBAL_EYEBROW_LG_SEL ) ),
+      globalEyebrowSM:   element( by.css( GLOBAL_EYEBROW_SM_SEL ) )
+    };
+  } );
+
+  beforeEach( function() {
+    browser.get( '/' );
+  } );
+
+  if ( browser.params.windowWidth > breakpointsConfig.bpLG.min ) {
+    describe( 'large size', function() {
+      describe( 'at page load', function() {
+        it( 'should include Header', function() {
+          expect( _dom.header.isDisplayed() ).toBe( true );
+        } );
+
+        it( 'should include logo', function() {
+          expect( _dom.logo.isDisplayed() ).toBe( true );
+        } );
+
+        it( 'should include Mega Menu', function() {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1754/commits/d9177fa74b300d92e11d1362c2e3627d5641c181